schema/google/showcase/v1beta1/echo.proto

```diff
@@ -43,6 +43,8 @@ service Echo {
   // This service is meant to only run locally on the port 7469 (keypad digits
   // for "show").
   option (google.api.default_host) = "localhost:7469";
+  // See https://github.com/aip-dev/google.aip.dev/pull/1331
+  option (google.api.api_version) = "moose";
```

Comment:
Ok and just to make sure.....we don't want a "real" version here?

I understand from generator/client perspective this is an opaque string, but there is something to be said about having good test data...

Reply:
Fixed in a37732121833e0a9d589611bb2b60b32b85d1479